src/components/pinnable.js

```diff
@@ -80,15 +86,15 @@ AFRAME.registerComponent("pinnable", {
     let didFireThisFrame = false;
     if (!held && this.wasHeld && isMine) {
       didFireThisFrame = true;
-      this._fireEventsAndAnimate(this.data, true);
+      this._fireEventsAndAnimate({ data: this.data, force: true });
```

Comment:
The function signature destructures `oldData` but this line creates an object with key `data`. Is this a bug?

Reply:
Oh yep, good catch. Updated.